File: src/primus/pawel/Exercise_136.java
package primus.pawel;

// Make a Window
// 6kyu
// https://www.codewars.com/kata/59c03f175fb13337df00002e/train/java
// 2021-05-25 T:01:12:00
class Exercise_136 {
    public static String makeAWindow(int num) {
        StringBuilder result = new StringBuilder();
        result.append("-".repeat(num * 2 + 3)).append("\n");

        result.append((("|" + ".".repeat(num)) + "|" + ".".repeat(num) + "|" + "\n").repeat(num));

        result.append("|").append("-".repeat(num)).append("+").append("-".repeat(num)).append("|").append("\n");

        result.append((("|" + ".".repeat(num)) + "|" + ".".repeat(num) + "|" + "\n").repeat(num));

        result.append("-".repeat(num * 2 + 3));
       // System.out.println(result.toString());

        return result.toString();
    }

    public static void main(String[] args) {
        makeAWindow(3);
    }

}

File: src/JS/Exercise_136.js
// Return the closest number multiple of 10
// 7 kyu
// https://www.codewars.com/kata/58249d08b81f70a2fc0001a4/train/javascript
// 2022-06-20 T:20:10:42

const closestMultiple10 = num => {
    return num % 10 < 5 ?
        num - num % 10
        : num + (10 - num % 10)
        ;
};